feder/monitorings/templates/monitorings/_tabs.html

```diff
@@ -0,0 +1,24 @@
+{% load i18n guardian_tags %}
+{% get_obj_perms request.user for object as "monitoring_perms" %}
+    <ul class="nav nav-tabs" role="nav">
+            <li
+                class="{% if tab == 'general' %}active{% endif %}"><a
+                {% if tab != 'general' %}
```

Comment:
Zastanawiam się czy to jest naprawdę konieczne sprawdzanie, czy nie rozwiązuje to samo ``class="active"``.

Reply:
Raczej nie chcemy linku (czy choćby kursora z łapką) na aktywnej zakładce?

Reply:
@jacekll , kursor jest zresetowany (zob. https://github.com/watchdogpolska/feder/blob/ce84da2d5a250104ab01ab861f52e305851c525a/feder/static/css/style.css#L2786-L2791 ).